src/Core/Geometry/TriangleMeshFactory.cpp

```diff
@@ -49,10 +49,11 @@ std::shared_ptr<TriangleMesh> CreateMeshSphere(double radius/* = 1.0*/,
 	mesh_ptr->vertices_[1] = Eigen::Vector3d(0.0, 0.0, -radius);
 	double step = M_PI / (double)resolution;
 	for (int i = 1; i < resolution; i++) {
+	    double alpha = step * i;
```

Comment:
Inconsistent indent.

Reply:
Fixed.

I prefer using **spaces** instead of **tabs**, though Open3D recommends using **tabs**.